pkg/skaffold/survey/config.go

```diff
@@ -69,7 +96,7 @@ func (s config) prompt() string {
 		return fmt.Sprintf(`%s: run 'skaffold survey'
 `, s.promptText)
 	}
-	return fmt.Sprintf(`%s: run 'skaffold survey -id %s'
+	return fmt.Sprintf(`%s: run 'skaffold survey --id %s'
```

Comment:
Filed #6198: we should be able to validate these commands in tests to catch these problems.